src/dom.js

```diff
@@ -530,6 +530,40 @@ export function getDataParamsFromAttributes(element, opt_computeParamNameFunc,
 }
 
 /**
+ * Returns link element url components as url parameters key-value pairs.
+ * e.g. <a href="https://example.com/index.html?query#hash"
+ * -> {
+ *   clickHostname: example.com,
+ *   clickProtocol: https,
+ *   clickProtocol: /index.html,
```

Comment:
clickProtocol here seems wrong.

Reply:
@avimehta Is it mean attaching colon? Since GTM deletes the colon, I think that it is better to make it the same.

Reply:
no i mean that it should be `clickPath` not `clickProtocol`

Reply:
Oh, I see. Sorry